code/game/machinery/alarm.dm

```diff
@@ -1047,8 +1047,7 @@
 		stat &= ~NOPOWER
 	else
 		stat |= NOPOWER
-	spawn(rand(0,15))
```

Comment:
I get what they were going for here---having a staggered effect of them turning off at random intervals, but....like, you're rarely going to notice this.

It's really fascinating to see the attention to detail some old coders had for small things like this, yet the detail to code quality was atrocious.

Reply:
Yeah, that's how I felt about this. It's like no one's going to care or notice this.